library/common/config/config.cc

```diff
@@ -95,6 +95,15 @@ R"(
   typed_config:
     "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
     common_tls_context:
+      tls_params:
+        tls_maximum_protocol_version: TLSv1_3
+        cipher_suites: ["ALL:!aPSK:!ECDSA+SHA1:!3DES"]
+        ecdh_curves:
+        - P-256
+        - X25519
+        # Non-default curves added that are used by Chrome
+        - CECPQ2
```

Comment:
This is an experimental code point and not one you should be enabling just yet. We ourselves are still in the process of rolling it in and out.
https://www.imperialviolet.org/2018/12/12/cecpq2.html